hack/istio/multicluster/start-minikube.sh

```diff
@@ -153,25 +172,13 @@ TOKEN_KEY=$(curl -k -X POST https://"${KUBE_HOSTNAME}"/realms/master/protocol/op
             -d scope=openid \
             -d response_type=id_token | jq -r '.access_token')
 
+# "https://_KIALI_KUBE_HOSTNAME/kiali/*`
 # Replace the redirect URI with the minikube ip. Create the realm.
-sed "s/_KIALI_KUBE_HOSTNAME/$(minikube ip -p "${CLUSTER1_NAME}")/g" "${SCRIPT_DIR}"/realm-export-template.json | curl -k -L https://"${KUBE_HOSTNAME}"/admin/realms -H "Authorization: Bearer $TOKEN_KEY" -H "Content-Type: application/json" -X POST -d @-
+jq ".clients[] |= if .clientId == \"kube\" then .redirectUris = [\"https://${CLUSTER1_IP}/kiali/*\", \"https://${CLUSTER2_IP}/kiali/*\"] else . end" < "${SCRIPT_DIR}"/realm-export-template.json | curl -k -L https://"${KUBE_HOSTNAME}"/admin/realms -H "Authorization: Bearer $TOKEN_KEY" -H "Content-Type: application/json" -X POST -d @-
```

Comment:
I'm confused what KUBE_HOSTNAME is... we have two minikubes here - which one does this variable represent? Should we rename this variable to MINIKUBE_HOSTNAME1 to make it clear this is the hostname of the first minikube  (which I assume it is?)

Reply:
Yeah it made more sense when there was just one. It's really the `KEYCLOAK_KUBE_HOSTNAME`. I'll rename it to that.